### PR TITLE
add submariner csv

### PR DESCRIPTION
**Pull request by skeeey**
see https://github.com/open-cluster-management/backlog/issues/6435

### CODE REVIEW

File: deploy/olm-catalog/manifests/lighthouse.submariner.io_serviceimports.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: serviceimports.lighthouse.submariner.io
```
Comment (qiujian16) [inline deploy/olm-catalog/manifests/lighthouse.submariner.io_serviceimports.yaml:5]: should we use upstream group?

File: deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs_crd.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (qiujian16) [inline deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs_crd.yaml:1]: where is  the type.go? I think we need at least some doc  on this  api.

File: deploy/olm-catalog/manifests/lighthouse.submariner.io_serviceimports.yaml (hunk 1 above)
Comment (skeeey) [reply deploy/olm-catalog/manifests/lighthouse.submariner.io_serviceimports.yaml:5]: not sure upstream group can work or not, I will try it

File: deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs_crd.yaml (hunk 2 above)
Comment (skeeey) [reply deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs_crd.yaml:1]: I will add it

Comment (skeeey) [reply deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs_crd.yaml:1]: submariner-operator 0.7.0 cannot support upstream api, we will update this later

File: deploy/olm-catalog/manifests/lighthouse.submariner.io_serviceimports.yaml (hunk 1 above)
Comment (skeeey) [reply deploy/olm-catalog/manifests/lighthouse.submariner.io_serviceimports.yaml:5]: submariner-operator 0.7.0 cannot support upstream api, we will update this later

File: pkg/apis/submarinerconfig/v1alpha1/types.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
```
Comment (qiujian16) [inline pkg/apis/submarinerconfig/v1alpha1/types.go:8]: should be namespaced

Comment (skeeey) [reply pkg/apis/submarinerconfig/v1alpha1/types.go]: removed

**Pull request merged**